pkg/sqlqueries/SQL/related_requests/related_intakes_by_intake_ids.sql

```diff
@@ -0,0 +1,49 @@
+WITH searched_intake_ids AS (
+  SELECT UNNEST(CAST(:system_intake_ids AS UUID[])) AS system_intake_id
+),
+related_intakes_by_system AS (
+	SELECT
+		sids.system_intake_id AS searched_intake_id,
+		-- sis_one.system_id AS related_system_id,
```

Comment:
can these commented out lines (and the ones below in this file + the other sql files) get removed?

Reply:
I left them in to help with debugging (uncommented they will return the matching related CEDAR system ID or contract number) but I can take them out.

Reply:
Removed in [69864fa](https://github.com/CMS-Enterprise/easi-app/pull/2678/commits/69864fa319bf4ca801e76f8ac80b8c3f187f66ed)